plugin/gcp/service.py

```diff
@@ -13,15 +13,32 @@
 #    * See the License for the specific language governing permissions and
 #    * limitations under the License.
 
-import time
 import json
+from functools import wraps
+import time
 
 import Crypto
 import httplib2
 from googleapiclient.discovery import build
 from oauth2client.client import SignedJwtAssertionCredentials
 
-from plugin.gcp import utils
+
+def blocking(default):
```

Comment:
Maybe there's no need for a default value here, and using the blocking decorator is indicative enough? (I see you always use "@blocking(True)" )